pages/homePage.js

```diff
@@ -0,0 +1,6 @@
+var homePage = function() {
```

Comment:
Currently you have homePage, loginPage and productPage. Each of these pages should have class extended from one main "Pages.js" file. The goal of the page object model is to reduce code duplication, so that you have all the methods within the page that can be inherited later.